templates/managed/index.html

```diff
@@ -17,17 +17,8 @@ <h1>Performant open source with Managed Apps</h1>
         <a href="/managed/contact-us" class="p-button--positive js-invoke-modal">Free deployment assessment</a>
       </p>
     </div>
-    <div class="col-6 u-hide--small u-align--center">
-      {{
-        image(
-        url="https://assets.ubuntu.com/v1/f880a3bd-Enterprise+support.svg",
-        alt="",
-        width="225",
-        height="225",
-        hi_def=True,
-        loading="auto",
-        ) | safe
-      }}
+    <div class="col-6 u-embedded-media u-hide--small u-align--center">
```

Comment:
```suggestion
    <div class="col-6 u-embedded-media u-align--center">
```

leave this for small screens